ui/src/app/views/registration-page/component.tsx

```diff
@@ -0,0 +1,183 @@
+    };
+    this.state.taskCompletionMap.set(0, props.trainingCompleted);
+    this.state.taskCompletionMap.set(1, props.eraCommonsLinked);
+  }
+
+  private registrationTasks = [
+    {
+      title: 'Complete Online Training',
+      description: 'Researchers must maintain up-to-date completion of compliance ' +
+        'training courses hosted at the NNLM\'s Moodle installation',
+      buttonText: 'Complete training',
+      completedText: 'Completed',
+      onClick: RegistrationPage.redirectToTraining
+    }, {
+      title: 'Login to ERA Commons',
+      description: 'Researchers must maintain up-to-date completion of compliance' +
+        ' training courses hosted at the NNLM’s Moodle installation',
+      buttonText: 'Login',
+      completedText: 'Linked',
+      onClick: RegistrationPage.redirectToNiH
+    }
+  ];
+
+  static redirectToNiH(): void {
+    const url = environment.shibbolethUrl + '/link-nih-account?redirect-url=' +
+            encodeURIComponent(
+              window.location.origin.toString() + '/nih-callback?token={token}');
+    window.location.assign(url);
+  }
+
+  static async redirectToTraining() {
+    await profileApi().updatePageVisits({page: 'moodle'});
+    window.location.assign(environment.trainingUrl + '/static/data-researcher.html?saml=on');
+  }
+
+  isEnabled(i: number): boolean {
+    const {taskCompletionMap} = this.state;
+    if (i < this.registrationTasks.length) {
+      if (i === 0) {
+        return !taskCompletionMap.get(i) && !taskCompletionMap.get(i + 1);
+      } else {
+        return !taskCompletionMap.get(i) && !taskCompletionMap.get(i + 1) &&
+          taskCompletionMap.get(i - 1);
+      }
+    } else {
+      return !taskCompletionMap.get(i);
+    }
+  }
+
+  allTasksCompleted(): boolean {
+    return Array.from(this.state.taskCompletionMap.values()).reduce((acc, val) => acc && val);
+  }
+
+  render() {
+    const {taskCompletionMap, trainingWarningOpen} = this.state;
+    const {betaAccessGranted, eraCommonsError, trainingCompleted} = this.props;
+    return <div style={styles.registrationPage}
+                data-test-id='registration-dashboard'>
+      <div style={styles.mainHeader}>Researcher Workbench</div>
+      <div style={{...styles.mainHeader, fontSize: '18px', marginBottom: '1rem'}}>
+        <ClrIcon shape='warning-standard' class='is-solid'
+                 style={{color: '#fff', marginRight: '0.3rem'}}/>
+        In order to get access to data and tools please complete the following steps:
+      </div>
+      {!betaAccessGranted && <div data-test-id='beta-access-warning'
+                                  style={{...baseStyles.card, ...styles.warningModal}}>
+        <ClrIcon shape='warning-standard' class='is-solid'
+                 style={styles.warningIcon}/>
+        You have not been granted beta access. Please contact help@research-aou.org.
+      </div>}
+
+      <div style={{display: 'flex', flexDirection: 'row', margin: '3%'}}>
```

Comment:
Nit: I think this looked better without the margin, we should check with Lou on that.

Reply:
mocks have it without margin so i will make this change